src/ucp/proto/rndv.c

```diff
@@ -74,8 +74,8 @@ size_t ucp_rndv_rts_pack(ucp_request_t *sreq, ucp_rndv_rts_hdr_t *rndv_rts_hdr,
     ucp_worker_h worker = sreq->send.ep->worker;
     ssize_t packed_rkey_size;
 
-    rndv_rts_hdr->sreq.reqptr = (uintptr_t)sreq;
     rndv_rts_hdr->sreq.ep_id  = ucp_send_request_get_ep_remote_id(sreq);
+    rndv_rts_hdr->sreq.req_id = ucp_send_request_get_id(sreq);
```

Comment:
we already have "worker" here, maybe use ucp_worker_get_request_id ?

Reply:
this local war will be compiled out in release build, IMHO `ucp_send_request_get_*id`  looks more consistent, WDYT?